fundingmanager.go

```diff
@@ -684,6 +719,12 @@ func (f *fundingManager) initFundingWorkflow(targetPeer *peer, req *openChanReq)
 // wallet, then sends a funding request to the remote peer kicking off the
 // funding workflow.
 func (f *fundingManager) handleInitFundingMsg(msg *initFundingMsg) {
+	if !f.checkMaxPending(msg.peer) {
```

Comment:
I think the check for max pending channels here should be removed. 

As suggested above, this option should only constrain the maximum number of _incoming_ pending channels, instead of cumulatively across both outgoing and incoming channels. 

Since this parameter is node policy, different nodes on the network may have a different set policy number. Therefore, as this policy isn't currently signaled anywhere, we must relay on a negative error message from the connected peer in order to properly limit the number of pending outgoing channels we make to that node.